### ISSUE TITLE
Default Accept header still sending application/xml?

### ISSUE THREAD

**Issue by dmcneil** [rest-client/rest-client#446]
I'm a bit confused/stuck on something related to the default `Accept` header that RestClient sends on requests.

I found this PR: https://github.com/rest-client/rest-client/pull/323 from a while ago that changes the default `Accept` header from `*/*; q=0.5, application/xml` to `*/*` but for some reason, when I do the following:
```
RestClient.get('some/resource')
```
It is still sending `*/*; q=0.5, application/xml` as the header. I'm using RestClient `1.8.0` which I believe is the latest but can't seem to figure it out. I also recreated the unit test located at the top (about line 26): https://github.com/rest-client/rest-client/blob/master/spec/unit/request_spec.rb but I seem to still be getting the wrong request `Accept` header...here was my test code:
```
@request = RestClient::Request.new(:method => :put, :url => 'http://some/resource', :payload => 'payload')
puts @request.default_headers[:accept]
puts Gem.loaded_specs["rest-client"].version # To verify my gem version...
```
And the output is:
```
*/*; q=0.5, application/xml
1.8.0 # Gem version
```
...I'm a bit lost here as to why it is still passing `application/xml` in the `Accept` header when looking through the source it shouldn't be doing that anymore. My gem seems to be the latest version, as well. Am I just missing something painfully obvious here? One thought that I had was that perhaps there was something going on with how it parsed the URI but I still couldn't find anywhere it would add `application/xml` in.

**Comment by dmcneil** [comment]
I was mistaken. `1.8.0` is not the latest gem version it would appear. I installed `2.0.0rc2` from https://rubygems.org/gems/rest-client/versions/2.0.0.rc2 and it seems the issue is fixed. The `Accept` header now returns just `*/*` as expected.

When I would do `gem install rest-client` it thought that `1.8.0` is the latest gem version and also on the main Github page for this the `GEM VERSION` icon says `1.8.0` as well. `gem update rest-client` also didn't do the trick...had to manually install `2.0.0.rc2`...which makes sense since all the `2.0.0` versions are still release candidates.

**Comment by ab** [comment]
Yeah that was only merged into the 2.0 series. Should be a final 2.0.0 release relatively soon.
